README.md

```diff
@@ -15,13 +15,111 @@
 
 ## Usage
 
-> Some usage instructions
+> Please run the following scripts to start this service:
+1. `npm run seed` : insert 400 mock items into database (business, photo)
+2. `npm run unseed`: remove all items from database
+
+## API Endpoint
+
+### 1. for business info display
+
+**URL:** /biz/:bId
+
+**Description:** This example retrieves current business detail of the specific business page with first 3 photos
+
+**API Response:**
+```
+  {
+    bId: 1,
+    bizname: 'Taste of Texas',
+    reviewCount: 1380,
+    rating: 4.5,
+    price: '$$$',
+    category: ['Steakhouses', 'Wine Bars'],
+    location: {
+      address1: '10505 Katy Fwy',
+      address2: 'Ste 2',
+      city: 'Houston',
+      state: 'TX',
+      zipcode: '77024',
+      neiborhood: 'Memorial',
```

Comment:
Nitpicking, but neighborhood is spelled incorrectly. This may lead to downstream problems.

Reply:
This could be a bug ...lol I will change it now! Thanks!!